inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -491,8 +492,14 @@ void GNADeviceHelper::open(uint8_t n_threads) {
 #else
     auto status = Gna2DeviceGetVersion(nGnaDeviceIndex, &detectedGnaDevVersion);
     checkGna2Status(status, "Gna2DeviceGetVersion");
-    status = Gna2DeviceOpen(nGnaDeviceIndex);
-    checkGna2Status(status, "Gna2DeviceOpen");
+
+    if (useDeviceEmbedded1_0) {
+        status = Gna2DeviceCreateForExport(Gna2DeviceVersionEmbedded1_0, &nGnaDeviceIndex);
```

Comment:
Reuse `GetDeviceVersionFromString`
```suggestion
        auto exportGeneration = GnaPlugin::GetDeviceVersionFromString(config.dumpXNNGeneration);
        status = Gna2DeviceCreateForExport(exportGeneration , &nGnaDeviceIndex);
```